zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -3341,6 +3341,72 @@ paths:
                 $ref: "#/components/schemas/ProblemDetail"
         "500":
           $ref: "#/components/responses/InternalServerError"
+  /resources/{resourceKey}:
+    get:
+      tags:
+        - Resource
+      operationId: getResource
+      summary: Get resource
+      description: |
+        Returns a deployed resource.
+        :::info
+        Currently, this endpoint only supports RPA resources.
+        :::
+      parameters:
+        - name: resourceKey
+          in: path
+          required: true
+          description: The unique key identifying the resource.
+          schema:
+            type: string
+      responses:
+        "200":
+          description: The resource is successfully returned.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ResourceResult"
+        "404":
+          description: A resource with the given key was not found.
+          content:
+            application/problem+json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetail"
+        "500":
+          $ref: "#/components/responses/InternalServerError"
```

Comment:
In the original PR the 500 is described in more detail. Is this expected?

Reply:
Yes, we streamlined some API responses on `main` in the OpenAPI spec for a more consistent API UX.